py-polars/polars/expr/expr.py

```diff
@@ -4657,16 +4657,19 @@ def rolling_min(
         this window will (optionally) be multiplied with the weights given by the
         `weight` vector. The resulting values will be aggregated to their sum.
 
-        If you pass a ``by`` column ``<t_0, t_1, ..., t_2>``, then by default the
-        windows will be:
+        If ``by`` has not been specified (the default), the window at a given row will
+        include the row itself, and the `window_size - 1` elements before it.
+
+        If you pass a ``by`` column ``<t_0, t_1, ..., t_2>``, then `closed="left"` means
+        the windows will be:
 
             - [t_0 - window_size, t_0)
             - [t_1 - window_size, t_1)
             - ...
             - [t_n - window_size, t_n)
 
-        Otherwise, the window at a given row will include the row itself, and the
-        `window_size - 1` elements before it.
+        With `closed="right"`, the first entry is not included and the last entry is
+        included.
```

Comment:
sorry to block, just thought that "the first entry" might not be clear, as the window might not include multiple entries

how about
```suggestion
        With `closed="right"`, the left endpoint is not included and the right endpoint is
        included.
```